lk2nd/device/dts/msm8996/msm8996-lg-h990.dts

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	model = "LG V20";
+	qcom,msm-id = <QCOM_ID_MSM8996 0x10000>;
+	qcom,board-id = <0xc08 0x00>;
+	qcom,pmic-id = <0x20009 0x10013 0x00 0x00>;
+};
+
+&lk2nd {
+	model = "LG V20 (International)";
```

Comment:
Could you please also add it to the devices.md list?

Reply:
I will close this pull request and create new pull request